agent/src/agent/pipeline/manager.py

```diff
@@ -318,7 +322,7 @@ def transform_for_bc(pipeline_: Pipeline) -> dict:
         'progress': {
             'last_offset': pipeline_.offset.offset if pipeline_.offset else '',
         },
-        'schema': pipeline_.get_schema() if pipeline_.get_schema_id() else schema.build(pipeline_),
+        'schema': pipeline_.get_schema(),
```

Comment:
what about pipelines that don't use schema yet?

Reply:
it will return empty dict. I guess we don't need to send schema for pipelines that don't actually use it

Reply:
it's the agent in bc script, the logic was that we build schema even if the pipeline is not using it. this way anodot UI can show dimensions and measurements

Reply:
really? Didn't know it, I thought that's a bug